scripts/Muon/GUI/Common/fitting_widgets/basic_fitting/fit_function_options_view.py

```diff
@@ -12,6 +12,7 @@
 from Muon.GUI.Common.utilities import table_utils
 
 from qtpy.QtWidgets import QWidget
+from qtpy import QtCore
```

Comment:
Can you move this above the `QWidget` import, and also only import `Qt` as this is all we need from `QtCore`
```suggestion
from qtpy.QtCore import Qt
from qtpy.QtWidgets import QWidget
```